tools/windows/DatadogAgentInstaller/WixSetup/Datadog/AgentInstaller.cs

```diff
@@ -50,99 +50,119 @@ public AgentInstaller()
+                    new RegistrySearch(
+                        RegistryHive.LocalMachine,
+                        "SOFTWARE\\Datadog\\Datadog Agent",
+                        "ConfigRoot",
+                        RegistrySearchType.raw)
+                )
+                {
+                    AttributesDefinition = "Secure=yes"
+                },
+                new CloseApplication(new Id("CloseTrayApp"),
+                    Path.GetFileName(_agentBinaries.Tray),
+                    closeMessage: true,
+                    rebootPrompt: false
+                )
+                {
+                    Timeout = 1
+                },
+                new RegKey(
+                    _agentFeatures.MainApplication,
+                    RegistryHive.LocalMachine, @"Software\Datadog\Datadog Agent",
+                    new RegValue("InstallPath", "[PROJECTLOCATION]") { Permissions = new[] { new Permission { User = "[DDAGENTUSER_DOMAIN]\\[DDAGENTUSER_NAME]", GenericAll = true } } },
+                    new RegValue("ConfigRoot", "[APPLICATIONDATADIRECTORY]") { Permissions = new[] { new Permission { User = "[DDAGENTUSER_DOMAIN]\\[DDAGENTUSER_NAME]", GenericAll = true } } }
+                )
+                {
+                    Win64 = true
+                },
+                new RemoveRegistryKey(_agentFeatures.MainApplication, @"Software\Datadog\Datadog Agent")
+            )
+            .SetProjectInfo(
+                upgradeCode: ProductUpgradeCode,
+                name: ProductFullName,
+                description: ProductDescription,
+                // SetProjectInfo throws an Exception is Revision is != 0
+                // we use Revision = 2 for the next gen installer while it's still a prototype
+                version: new Version(_agentVersion.Version.Major, _agentVersion.Version.Minor,
+                    _agentVersion.Version.Build, 0)
+            )
+            .SetControlPanelInfo(
+                name: ProductFullName,
+                manufacturer: CompanyFullName,
+                readme: ProductHelpUrl,
+                comment: ProductComment,
+                contact: ProductContact,
+                helpUrl: new Uri(ProductHelpUrl),
+                aboutUrl: new Uri(ProductAboutUrl),
+                productIconFilePath: new FileInfo(ProductIconFilePath)
+            )
+            .DisableDowngradeToPreviousVersion()
+            .SetMinimalUI(
+                backgroundImage: new FileInfo(InstallerBackgroundImagePath),
+                bannerImage: new FileInfo(InstallerBannerImagePath),
+                // $@"{installerSource}\LICENSE" is not RTF and Compiler.AllowNonRtfLicense = true doesn't help.
+                licenceRtfFile: new FileInfo(ProductLicenceRtfFilePath)
+            )
+            .AddDirectories(
+                CreateProgramFilesFolder(),
+                new Dir(new Id("APPLICATIONDATADIRECTORY"), @"%CommonAppData%\Datadog",
```

Comment:
I can't find MSDN docs on `%CommonAppData%`. Some forums make it seem like a legacy var from when the program data path was `C:\Documents and Settings\All Users\Application Data\`.
I don't have the env var on my host, we use `%ProgramData%` instead.